notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p><strong>If you don't already have a Cloud Storage bucket that you can use</strong>, run the following cell to create one. The bucket will be deleted in the <strong>Cleaning up</strong> section at the end of this tutorial. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>